Zend/zend_API.c

```diff
@@ -40,6 +40,8 @@
 /* these variables are true statics/globals, and have to be mutex'ed on every access */
 ZEND_API HashTable module_registry;
 
+bool use_deepbind = false;
```

Comment:
This must be ZEND_API to use the symbol within DL_LOAD() from extensions.

Reply:
I'd call it also zend_dl_use_deepbind. The reason being that public symbols normally need to be prefixed by zend_ or php_ (depending on where it is) and the "dl" in the prefix makes it clear it's about dynamic loading.